3529/2/selfie_handler.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'image_file_writer'
+
+module SelfieHandler
+  include ImageFileWriter
+
+  def selfie_handler(path, in_or_out)
+    path_img = path.create_directory(in_or_out, "#{payload['from']['username']}.jpg")
```

Comment:
And why you need to create a directory here? and as I see `path` is not a path to file or etc it's object of `PathLoader`

Reply:
as far as i understend task, we need to do it, 'cause with this prog we find names of directory (we need to know session key and time)
or i'm confused